### PR TITLE
[tests] improve fuzz test, fuzz corpus test runner

### PR DESCRIPTION
**Pull request by tmc**
This improves the fuzz test corpus runner and is related to changes from #80 and can mark #78 as closed.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -48,7 +48,6 @@ testv:
 
 coverage:
 	${GOBIN} test -covermode atomic -coverprofile cover.out ./src/sybil
-	sed -i "s|_${ROOT_DIR}|.|"	cover.out
```
Comment (okayzed) [inline Makefile]: this is for me and my non standard GOPATH, don't remove please

File: Makefile [hunk 2]
```diff
@@ -84,6 +83,10 @@ fuzz: bin/cmd-fuzz-query.zip
 fuzzv: bin/cmd-fuzz-query.zip
 	FUZZDEBUG=1 go-fuzz -bin=./bin/cmd-fuzz-query.zip -workdir=workdir/query -testoutput -procs 1
 
+fuzzcoverage:
```
Comment (okayzed) [inline Makefile]: fails on go 1.6 for me (with test.Run in fuzz_test.go)

Comment (tmc) [reply Makefile]: I'll ad a build tag.

File: Makefile (hunk 1 above)
Comment (tmc) [reply Makefile]: I think it was messing up codecov reports, I'll remove for now and investigate more.

**Pull request merged**